apis/models.d.ts

```diff
@@ -32,7 +32,7 @@ type filename = string
  * The effective CDS model loaded during bootstrapping, which contains all service and entity definitions,
  * including required services.
  */
-export const model : LinkedCSN | undefined  // was ?: LinkedCSN
+export let model : LinkedCSN | undefined  // was ?: LinkedCSN
```

Comment:
`cds.model` got set by the test below, so we consequently need to commit to a settable property here.

Reply:
Good point! Could we keep this is an inline comment as reminder for our future selfs?
```suggestion
// cds.model can be set, so it can not be const
export let model : LinkedCSN | undefined  // was ?: LinkedCSN
```

Reply:
I have added a comment.